Libs/MRML/Core/vtkMRMLDoubleArrayNode.h

```diff
@@ -100,7 +100,16 @@ class VTK_MRML_EXPORT vtkMRMLDoubleArrayNode : public vtkMRMLStorableNode
   /// int GetXYAxisValue(int index, double* x, double* y, double*  yerr);
 
   ///
-  /// Set X and Y values at he data point specified by 'index'
+  /// Set values at the data point specified by 'index'
```

Comment:
Even better, you are right, it makes sense to keep the notion of data point. Forget my comment. What you wrote make perfect sense.